docs/development.md

```diff
@@ -55,10 +54,42 @@ The box can now be accessed via ssh and the Rails server started directly (this
 ```sh
 vagrant ssh centos7-devel
 cd foreman
-bundle exec foreman start
 ```
 
-The Foreman UI is then accessible on `https://centos7-devel.<hostname>.example.com/`, where `<hostname>` is the shortname of your workstation.  While using the webpack dev server, you must use HTTPS and the hostname, as well as visiting `https://centos7-devel.<hostname>.example.com:3808` to accept the self-signed certificate.
+### Starting the development server
+
+Our backend requires a rails server to be running. We also use webpack server for the more "modern" parts of our frontend UI. 
+[Webpack](https://webpack.js.org/) is a way of bundling up multiple front-end files and assets to a small amount of files to send to the browser. 
+The files that webpack handles are located in `webpack/` directory found in Foreman, Katello,
+and plugin root directories. If you are editing any files in `webpack/`, and want to have your changes refresh automatically, you will need a webpack server running.
+
+Because we are using the webpack server in conjunction with a rails server, there are different ways of starting a server depending on your needs and preferences. The following are the
+instructions for starting the server using a base `centos7-devel` box as a starting point.
+
+#### Run a rails and webpack server together using `foreman start`
+- run `foreman start` in `~/foreman`
+- navigate to `https://centos7-devel.<hostname>.example.com/` (or use your prefered way of navigating your dev server).
+- accept the self-signed certs at `https://centos7-devel.<hostname>.example.com:3808`.
+  - If you are doing remote development, you will have to forward port 3808 from devel VM to your hypervisor. i.e. in forklift directory,
+    run `vagrant ssh centos7-devel -- -L 0.0.0.0:3808:localhost:3808`
+- Everything should be set for you to run `foreman start` to start your dev server as needed.
+
+NOTE: the `foreman` in `foreman start` is actually [this gem](https://github.com/ddollar/foreman) and not our `foreman`. It
+doesn't allow STDIN, which means it doesn't support debuggers like `byebug` and `pry` very well. To support debuggers, run the rails
+server and webpack separately as described below.
+
+
+#### Run the rails server without a webpack server running
+If you don't need to do any development in the `webpack/` directories, you can turn the wepback server off and just run the rails server if desired.
+
+- Change ':webpack_dev_server:` to `false` in `~/foreman/config/settings.yaml`
+- run `rake webpack:compile` in `~/foreman`, this bundles the assets that the webpack server would manage.
+- start the rails server with `rails s` in `~/foreman`
+  - use `rails s -b 0.0.0.0` if you are doing remote development
```

Comment:
Wouldn't it make sense to just have users do `source .env; rails s` instead, in case .env later contains things other than -b needed to start rails?

Reply:
Oh, @ekohl's suggestion is better: `bundle exec foreman start rails`